Modules/Journey/QuestieJourney.lua

```diff
@@ -112,6 +113,10 @@ function QuestieJourney:BuildMainFrame()
                 text = l10n("Quests by Zone"),
                 value="zone"
             },
+            {
+                text = l10n("Quests by Faction"),
```

Comment:
This string also needs to be added to the translation files. See `Localization/Translations/Journey`